tox.ini

```diff
@@ -17,7 +17,7 @@ deps =
     pytest-flakefinder
 install_command = 
     py{27,35,36,37,38,39}: pip install -f https://download.pytorch.org/whl/torch_stable.html {opts} {packages}
-    pylaunch: pip install -f https://download.pytorch.org/whl/torch_stable.html docker chardet iso8601 jupyter-repo2docker {opts} {packages}
+    pylaunch: pip install -f https://download.pytorch.org/whl/torch_stable.html docker chardet iso8601 yaspin jupyter-repo2docker {opts} {packages}
```

Comment:
Would be nice to have a `pip install wandb[all]` flavor that tox could use here.  Also we don't need the docker package.

Reply:
Removing the docker package.
Similar to the nbconvert ticket, I could put these into requirements_dev.txt for now.